Main.java

```diff
@@ -0,0 +1,50 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+    static String scanLine() {
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+    public static void main(String[] args) {
+        int start, end, length, var, aux;
+        int mean = 0;
+        System.out.println("Arithmetic mean algorithm");
+        System.out.println("""
+            Write 1 or 2 depending the case:\s
+            1 - arithmetic mean of a certain number range\s
+            2 - arithmetic mean of a certain count of numbers""");
+        var = Integer.parseInt(scanLine());
+        switch (var) {
+            case 1 -> {
+                System.out.println("Write the initial start of the range");
+                start = Integer.parseInt(scanLine());
+                System.out.println("Write the end of the range");
+                end = Integer.parseInt(scanLine());
+                if(start > end){

```

Comment:
start si end sunt locale aici. Au sens in blocul asta sa fie declarate, nu la nivelul intregii functii. La fel si la length.